### ISSUE TITLE
Feature request: send to Chromecast

### ISSUE THREAD

**Issue by pataquets** [KiraLT/torrent-stream-server#90]
It would be all awesome and cool function to add a button besides stream an download.

Although there seems to be a handful of Chromecast libs around, looks like it it would be doing some gluing to get it. However I can't tell for sure, and it might be easier said than done.

However, while researching about it I've found this code which might be useful (I can't judge it myself due to my poor JS): https://github.com/asapach/peerflix-server/compare/master...stevetuk71:master
It's using the https://chromecast.wyrt.co.uk/ service, which looks like way easier to implement.

**Comment by omgbox** [comment]
> It would be all awesome and cool function to add a button besides stream an download.
> 
> Although there seems to be a handful of Chromecast libs around, looks like it it would be doing some gluing to get it. However I can't tell for sure, and it might be easier said than done.
> 
> However, while researching about it I've found this code which might be useful (I can't judge it myself due to my poor JS): [asapach/peerflix-server@master...stevetuk71:master](https://github.com/asapach/peerflix-server/compare/master...stevetuk71:master)
> It's using the https://chromecast.wyrt.co.uk/ service, which looks like way easier to implement.

you can cast to chrome many ways

Browser , while video is playing right click and click on CAST option. chrome browser. -- > chromecast device
using VLC , click on playback ---> renderer --- > chromecast device 
there are many commandline tools, i use https://pypi.org/project/catt/ very easy tool. 

probably good idea to have player chromecast support but, right now i believe developer would need to sign to google app ID and pay small fee for the chromecast app.  long process it seems. 

the methods i listed are pretty straight forward and easy

**Comment by KiraLT** [comment]
Added Chromecast support to the video player.

![Screenshot from 2021-06-01 11-47-46](https://user-images.githubusercontent.com/10944908/120294787-38174500-c2cf-11eb-892c-faad88922e16.png)

**Comment by omgbox** [comment]
that looks very interesting , going to test it

**Comment by pataquets** [comment]
Awesome, @KiraLT !! :rocket: Thanks for adding this!
I'll test it as soon as I get to my Chromecast and report back.
Actually, I've tried serving from my machine's TSS instance a torrent to be casted on a CC over the Internet (via Cloudflare), but failed at it. However, I tried it with a remote user and I've still not ruled out some fault on my part (just in case you have any useful advice).
Remote user was casting from an Android phone to a CC on the same LAN. I'll have to look better at it and do a few more tests.
